bin/configure

```diff
@@ -64,7 +64,10 @@ end
 
 # Unless the shell's current version of Ruby is the same as what the application requires, we should flag it.
 # rbenv produces strings like "3.1.2" while rvm produces ones like "ruby-3.1.2", so we account for that here.
-required_ruby = `cat ./.ruby-version`.strip.gsub(/^ruby-/, "")
+asdf_version = `cat ./.tool-versions`.lines.find { |i| i.include?("ruby") }.gsub(/\b\w*[ruby]\w*\b/i, "").strip
+rvm_version = `cat ./.ruby-version`.strip.gsub(/^ruby-/, "")
```

Comment:
Can we potentially leave the variable name here as `required_ruby`, or maybe use a switch statement for the different types of version managers? I personally use rbenv, and `.ruby-version` only has the version number (i.e. - `3.2.2`) which shouldn't have any bearing on `gsub(/^ruby-/, "")`. What do you think?